src/ert/_c_wrappers/job_queue/run_status.py

```diff
@@ -1,36 +1,48 @@
 from dataclasses import dataclass
-from typing import Optional
+from enum import Enum
 
 
-@dataclass
-class RunStatus:
+class StatusEnum(Enum):
```

Comment:
I think maybe name the Enum `RunStatus` and the dataclass `RunResult`.

Reply:
I am not sure about the suggested names, can you give me a few reasons why renaming these would make things better?

Reply:
The name currently has two non-descriptive parts: `Status` which could be anything (what is it the status of?) and `Enum` which just reiterates the type.

Reply:
Hmm how about renaming `RunStatus` to `RunMonitor` or `RunTracker`, because that is the main purpose of the class keeps track or monitors the state of a run also renaming `StatusEnum` to `RunStatus`.

Reply:
Perfect!